server/core/src/initServer.cpp

```diff
@@ -875,6 +875,11 @@ initRsCommWithStartupPack( rsComm_t *rsComm, startupPack_t *startupPack ) {
         }
         rsComm->irodsProt = ( irodsProt_t )atoi( tmpStr );
 
+        if (rsComm->irodsProt != NATIVE_PROT && rsComm->irodsProt != XML_PROT) {
```

Comment:
`rsComm->irodsProt`'s value is obtained from the environment variable `SP_PROTOCOL`.

Does anyone know if `SP_PROTOCOL` serves the same purpose as `irodsProt`?